### PR TITLE
Order resume with payment info in it 

### PR DESCRIPTION
**Pull request by julianoviedo1**
This PR still needs work.

image

### CODE REVIEW

**Comment by julianoviedo1** [comment]
Added order resume and orderconfirmation:

image

image

File: app/javascript/packs/ordersForm.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import flatpickr from "flatpickr";
+
+$(document).ready(function() {
+
```
Comment (jpbalarini) [inline app/javascript/packs/ordersForm.js]: we usually don't leave this blank space here

File: app/javascript/packs/ordersForm.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    $('#delivery').addClass('hovered');
+  })
+  
+  $('#shop').on('click', () => {
+    $('#homeDelivery')[0].checked = true;
+    $('#delivery').removeClass('hovered');
+    $('#shop').addClass('hovered');
+  })
+
+  $("#deliveryDate").flatpickr({
```
Comment (jpbalarini) [inline app/javascript/packs/ordersForm.js]: use double quotes or single quotes, please don't mix both

File: app/models/payment.rb [hunk 3]
```diff
@@ -8,7 +8,23 @@ class Payment < ApplicationRecord
...
+    active.first.current_method = false
+  end
+
+  def current_method?
+    current_method
+  end
+
+  def type_info
+    if %w[credit_card debit_card].include? payment_type
+      card_number.to_s.last(4).to_i
```
Comment (jpbalarini) [inline app/models/payment.rb]: what happens if my CC ends in 0021? to_i is going to display 21 right? Maybe the to_i is not necessary

File: app/views/orders/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<header>
+  <%= render 'shared/navbar' %>
+</header>
+<section>
+  <%= render  'form' %>
```
Comment (jpbalarini) [inline app/views/orders/edit.html.erb]: good 👍

**Review by jpbalarini — CHANGES_REQUESTED** [review]

File: app/models/payment.rb (hunk 3 above)
Comment (julianoviedo1) [reply app/models/payment.rb]: Commit: b5b13dd

File: app/javascript/packs/ordersForm.js (hunk 2 above)
Comment (julianoviedo1) [reply app/javascript/packs/ordersForm.js]: Commit: b5b13dd

File: app/javascript/packs/ordersForm.js (hunk 1 above)
Comment (julianoviedo1) [reply app/javascript/packs/ordersForm.js]: Great, thanks! Commit: b5b13dd

**Review by jpbalarini — APPROVED** [review]

**Pull request merged**